luxonis_ml/scheme/v1/config_building_blocks/base_models/head.py

```diff
@@ -0,0 +1,192 @@
+from abc import ABC
+from typing import List, Optional, Union
+
+from pydantic import BaseModel, Field, validator
+
+from ..enums import ObjectDetectionSubtypeYOLO
```

Comment:
These seem like invalid imports

Reply:
Thanks. A bunch of files got screwed up in the merge